docs/README.md

```diff
@@ -201,6 +201,27 @@ versions:
   for Linux
 - [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows
 
+##### Install Git
+To initialize this project in your machine you need to install git 
+
+###### Ubuntu
+
+```shell
+sudo apt install git-all
+```
+
+###### macOS
+
+```shell
+brew install git
+```
+
+###### Windows
+
+```shell
+choco install git
+```
+
```

Comment:
Could you add a comment here regarding that the user has the `user.email` and `user.name` properly configured globally?

Reply:
I make some changes